apps/s_client.c

```diff
@@ -1424,6 +1424,13 @@ int s_client_main(int argc, char **argv)
     if (argc != 0)
         goto opthelp;
 
+#ifndef OPENSSL_NO_NEXTPROTONEG
+    if (min_version == TLS1_3_VERSION
+        && max_version == TLS1_3_VERSION && next_proto_neg_in != NULL) {
+        BIO_printf(bio_err, "Cannot supply NPN with TLSv1.3\n");
+        goto end;
```

Comment:
OK. But it might not be that consistent, for what in s_server, same errors of this type are not goto opthelp. I suppose those should be also corrected later.